ansys/mapdl/core/mapdl_functions.py

```diff
@@ -2008,7 +2008,7 @@ def pdlhs(self, nsim="", nrep="", isopt="", astop="", accmean="",
         will terminate the simulations before the NSIM*NREP
         simulations are done if the convergence criteria are met.
         """
-        command = f"PDLHS,{nsim},{nrep},{isopt},,{astop},{accmean},{accstdv},{check},{seed}"
```

Comment:
and this one was a spelling mistake I noticed. This one might need changing back? But it does read `accstdev` in the docstring so I assumed it was also an error

Reply:
Just checked 2021R1 MAPDL docs and it appears this command has been deprecated.  Let's just remove it.

Reply:
shall do!